contracts/proxy/ERC1967/ERC1967Utils.sol

```diff
@@ -137,13 +137,6 @@ library ERC1967Utils {
     // solhint-disable-next-line private-vars-leading-underscore
     bytes32 internal constant BEACON_SLOT = 0xa3f0ad74e5423aebfd80d3ef4346578335a9a72aeaee59ff6cb3582b35133d50;
 
-    /**
-     * @dev Returns the current beacon.
-     */
-    function getBeacon() internal view returns (address) {
-        return StorageSlot.getAddressSlot(BEACON_SLOT).value;
-    }
-
```

Comment:
I'd keep that as a primitive if anyone wants to implement a storage based beacon.